src/syntax/FStar.Syntax.Print.fs

```diff
@@ -583,7 +583,7 @@ let term_to_string' env x =
 
 let binder_to_json env b =
     let (a, imp) = b in
-    let n = if is_null_binder b then JsonNull else JsonStr (imp_to_string (nm_to_string a) imp) in
+    let n = if is_null_binder b then JsonStr "_" else JsonStr (imp_to_string (nm_to_string a) imp) in
```

Comment:
I think we could just drop the `if` and do:
```fstar
let n = JsonStr (imp_to_string (nm_to_string a) imp) in
```
which would be a bit cleaner, but also print the bv's id when `--print_full_names` is on.